mbed_app.json

```diff
@@ -33,6 +33,13 @@
             "nsapi.default-cellular-apn": 0,
             "nsapi.default-cellular-username": 0,
             "nsapi.default-cellular-password": 0
+        },
+        "DISCO_L496AG": {
+            "target.macros_add": [
+                "CELLULAR_DEVICE=STModCellular"
```

Comment:
I guess `CELLULAR_DEVICE` macro is not needed.

Reply:
I agree... 
CELLULAR_DEVICE is not used at all...?